app/templates/join.hbs

```diff
@@ -1,11 +1,24 @@
 {{page-title "Join"}}
 
 <div class="container mx-auto lg:max-w-screen-lg py-16 pb-48 px-3 md:px-6">
-  <h1 class="text-2xl font-bold text-gray-700 md:text-3xl text-center">
-    Join the best.
-  </h1>
+  <AffiliateLinkPage::HeroSection />
+  <AffiliateLinkPage::AcceptReferralContainer @affiliateLink={{this.model.affiliateLink}} class="mt-20 md:mt-24 mb-52" />
 
-  <AffiliateLinkPage::AcceptReferralContainer @affiliateLink={{this.model.affiliateLink}} class="mt-8" />
+  <div class="space-y-6 text-center my-8 max-w-[640px] max-h-[198px] mx-auto">
```

Comment:
Max width is fine, is there a reason we're doing max height too? What happens if the max-height rule isn't present? Using "magic" numbers like `198px` is usually a red flag suggesting that there's a more "responsive" way to implement things that doesn't rely on such numbers.